### DISCUSSION TITLE
你到底是I人還是E人呢？這真的準確嗎？

### DISCUSSION THREAD

**Discussion by thz0000** [1101-GS4719-C/Python-Community-GS4719-C#1712] [General]
近期許多心理測驗如雨後春筍般冒出，最廣為人知的莫過於「MBTI16型人格測驗」。它將人的性格分為：外向/內向、實感/直覺、理性/感性、系統/彈性。其中最令人津津樂道的就是外向型（extraversion）和內向型（introversion），也就是如今俗稱的E人和I人，或者社牛和社恐。

那這個測驗的是不是真的能代表一個人的性格呢？我認為並不完全是。

生活中，社交是不可避免的一道課題。有些人測驗出來的結果為內向型人格，但為了融入群體、更好地和大家相處，平常的表現卻是嘻嘻哈哈、樂於展現自己，那他到底是屬於內向還是外向呢？說內向吧，但表現出來完全不是這一回事；說外向吧，但測驗出來卻也不是。所以我想這個問題見仁見智。

而人的性格是會隨時間和環境變化的。有些人可能生性樂天、熱愛與人交流，卻因突逢變故而封閉自我、冷淡寡言；有些人或許原本不喜社交、沉默少言， 但在朋友的帶領下日漸打開自己，變得活潑開朗。

那MBTI也能如實呈現以上的轉變嗎？我覺得這是個未知數。

總而言之，心理測驗或許能簡易地分類人的性格，但更詳細、更複雜的情形是否同樣適用，還有待商榷。

**Comment by Yanxxx7979** [comment]
我覺得這些種類繁多的心理測驗確實能反映出一個人的某一面，因此大家會覺得很準；但畢竟我們本來就是相當複雜的生物，甚至每分每秒的心情狀態都不一樣，因此我認為它的確有它的原理，也能反映出人的某一面，有它的準確度存在，但若是盡信，就沒有必要了

**Comment by THH1118** [comment]
最近有看到一種說法是E是向外得到能量，I就是從內獲得能量，但我自己感覺其實結果是I或者E都不是100%的，只是其中一種的比例較多，所以才說MBTI是會改變的。

**Reply by woohsin** [reply]
我覺得真的會變，剛上大學的時候測，結果大二就變了，不知道經歷了甚麼哈哈。

**Comment by 12Jesse24** [comment]
我覺得這種測驗，今年測跟明年測出來的結果可能會不一樣，就像樓主說的人是會隨時間、環境甚至是發生的重大事故而有所改變，因此我覺得這種測驗只可能反映出近幾個月來的性格，若把時間放久遠來看則無法反映出人所改變的一面。

**Comment by mrweixd** [comment]
我以為區分Extroversion和 Introversion 的重點在於恢復能量的方式，跟個人社交能力沒關係，後者是可以學習的。內向者代表著是可以藉由獨處來恢復能量，他們會需要時間與自己相處，而不是代表著不會社交的人。

**Comment by uwje** [comment]
我認為這種測驗僅能作為一個參考，讓你跟認識自己是較外向或內向的性格，但性格的確會隨著時間改變而變化的，也沒必要強迫自己去成為誰

**Comment by sorn32** [comment]
我覺得這個測驗，比起給自己貼上標籤，更應該是想要用一些問題集，了解現在自己對自己的看法 ｡:.ﾟヽ(*´∀`)ﾉﾟ.:｡

**Comment by 111426027** [comment]
對我而言，我覺得測驗結果蠻符合的，但是隨著時間測驗結果也會不同，所以這種測驗可以作為參考就好。

**Comment by chuanmin0317** [comment]
我覺得這種測驗只是一種認識自己的方式，它能夠指出我們可能的缺點，讓我們能往自己想要的方向邁進

**Comment by YifenLin** [comment]
心理測驗真的是搞不懂，有時測出來的結果完全不像自己。我覺得人的性格是會受到各種因素影響的，不只是固定的。有時候我可能表現得很外向，但心裡其實挺內向；有時候又可能相反。所以，那些測驗只是參考，不能完全代表你的個性。開心就好，不必太在意。

**Comment by SusuTima** [comment]
我認為內向外向是取決於我處在的環境和遇到的人，所以我個人不太看重測驗結果，活得舒適比我是內向還是外向的人更為重要。

**Comment by 20050201Shawn** [comment]
之前做mbti的測驗，其實感覺蠻結果論的，就是選的哪種傾向比較多就偏向哪種人，但如果過於明確分清I跟E人可能有些偏頗，畢竟人是複雜的，個性的某一面(例如內外向)不是非黑即白，比較像是一種光譜。如上幾樓所提的，當作參考或好玩就好。

**Comment by YANRONG0123** [comment]
我認為這個測驗，只能檢測到施測者當下的狀態，可能隔天遇到某一個事件，進而影響到心情，就會改變測驗結果，畢竟很少事情是能夠保持一成不變的。

**Comment by happy246810** [comment]
對於我來說，我的測驗結果與我的真實情況還蠻相符的。但我相信這項測驗的結果可能跟每個人的真實情況都不盡相同，因為測試者當下的心情也會影響測試結果。所以其實也不用太在意結果，他只是一個參考。

**Comment by LinChloe** [comment]
我個人覺得mbti本來就只有比較偏向哪方面。
當時做測試時它都會附上你的%數。
所以我都這麼認為
I (70%)
在面對事情時，會有70%你會偏內向。
但大家現在都只說你是I還是E，才導致大家說它很二分法。

**Comment by yxin214** [comment]
本身並不覺得這種測驗是絕對的，但每次遇到心情不好或有點迷失自己的時候就很愛做這種心理測驗，我覺得他一定程度上能幫助自己分析現在的狀態等，還是有參考價值的。

**Comment by tzurou** [comment]
雖然知道這些測驗只是參考用的並非絕對，但是有一個測驗能夠更了解自己的性格和別人的差別對我來說蠻有用的，比較容易用對方的角度來思考為甚麼他會這麼做，每隔一段時間再做也能了解自己的性格是不是有改變，反正我自己就蠻喜歡這種測驗的

**Comment by yyujimmy** [comment]
你提出的觀點很中肯。確實，心理測驗如MBTI等提供了一種分類性格的方法，但它們並不是完美無缺的。性格是一個相當複雜的概念，受到許多因素的影響，包括環境、社會文化、個人經歷等等。

對於一個人的性格，單一的測驗結果可能無法涵蓋所有的情況。比如，有些人可能在特定情況下表現得外向，而在其他情況下可能表現得內向。而且，人的性格也會隨著時間和經歷而發生變化，這就增加了準確性格分類的難度。

因此，我們在使用心理測驗時，應該謹慎對待，並結合其他相關信息進行綜合評估。此外，我們也應該意識到，性格並不是一成不變的，而是可以發展和改變的。

**Comment by willychiuchiu** [comment]
這種心理測驗雖然提供了一個初步了解人格的工具，但人的性格是複雜且變化的，可能在不同環境或不同時間下測驗都會有不同的結果，所以我們應該以開放的態度看待，真正和一個人相處，才能更全面地了解一個人。

**Comment by sunset513** [comment]
我自己是蠻相信這個測驗結果的，我是ENFJ-T，我覺得自己的表現大致上吻合這個類型。

另外我發現一個有趣的點，就是我的生活圈中的好朋友們的人格大都和我蠻相似的，挺神奇的。


總結來說，我覺得這個測驗還蠻有跟參考功能的，感謝樓主的分享。

**Comment by tshsu04224** [comment]
我個人玩過不少次MBTI的測驗，我認為這種分類方式雖然或許比過往其他的心理測驗來得嚴謹，但卻也不見得比較準確，以我的例子來說，許多問題的選項可能我並沒有辦法很直覺判斷自己一定會這麼做，但還是硬選了一個答案出來，可想而知的最後數據就很模稜兩可，兩個面向總是40幾趴對50幾趴的數字，而且每次做每次結果都不太一樣。該說這種心理測驗的分類有點太極端嗎，每個人的性格或多或少都不盡相同，對待每件事的態度也可能不一樣，這樣把人分類分到不同的區塊，即使可能知道自己傾向，但我相信每個區塊的人還是有很大區別的。

**Comment by Eileen050824** [comment]
我自己有測過4次MBTI(都是因為各種作業需求強迫測的)，但我每一次測出來全都是ISTJ(物流師那個)，可能我的人格特質很固定吧！不過我身邊有朋友隨著環境不同，MBTI有180度大轉變，所以我相信本文所述為實，但我覺得只要自己自在就好！

**Comment by William08290829** [comment]
我覺得心理測驗可以提供一定程度上的人格特徵分類，但這些分類並不是絕對準確的。許多心理測驗，如MBTI、大五人格等，都試圖根據被測者的回答，將其分類為某種性格類型或特徵。這些測驗通常基於心理學理論和研究，具有一定的科學基礎，可以提供對個人性格的大致了解。但心理測驗也存在一些限制和局限性。人的性格是非常複雜和多樣化的，單一的測驗往往難以全面評估一個人的性格。還有測試結果可能受到受試者主觀因素、測試環境等因素的影響，存在一定的主觀性和偏差。

**Comment by Hahahahbla** [comment]
「MBTI16型人格測驗」近年颇受关注，尤其外向和内向类型。然而，它未必完全代表个体性格，因为人的性格会随时间和环境变化。测试结果可能不符合个体行为表现，因此其准确性有待商榷。至于外向和内向类型的人，外向者可能在社交場合中更活躍，能夠輕松地與他人建立聯繫，而內向者則可能更擅長深度思考和獨立工作，具備一定的自我反省能力

**Comment by jim1743** [comment]
我認為這種心理測驗，是反映了自身近期的心理狀態，讓我們更加認識自己是個怎樣的人，我認為對於青少年很有幫助，因為他們處於還在尋找自我的時期，有了這項測驗當作參考，能夠幫助他們更加了解自己。

**Comment by cassy0201** [comment]
這個測驗是會隨著自身經歷改變的，沒有絕對性，但能反映最近的身心狀態以及行事風格，幫助大家了解近期的自己

**Comment by stone971275** [comment]
MBTI（Myers-Briggs Type Indicator）是一種人格測試工具，用於協助了解個人的人格特質和偏好。它主要基於卡爾·荣格的心理學理論而發展出來，透過四個維度對人格進行分類，分別是外向(E)或內向(I)、感覺(S)或直覺(N)、思考(T)或情感(F)、判斷(J)或知覺(P)。這些分類可以形成16種不同的人格類型。

然而，MBTI被商業化並廣泛應用後，有時確實會被人們誤解或誤用，甚至被拿來當成藉口或標籤來解釋個人行為。甚至到達像星座一樣，有時候人們可能會將MBTI的結果視為一種“預言”或“解釋”，而非僅僅是了解自己或他人的一種方式。

關於I/E這個維度，確實與個人是否有能力面對人群或陌生人並沒有直接的關係。外向（E）和內向（I）只是指個人的能量取向，外向者喜歡外在世界和人際互動，內向者則更喜歡內在思考和獨處。一個外向者不一定就有能力面對人群，同樣一個內向者也可能是個出色的公眾演說者。因此，MBTI僅供參考，不應被過度解讀或濫用。

**Comment by 710388** [comment]
這個東西每次測都不太一樣，不好玩

**Comment by OHoln** [comment]
這種心理測驗我一直都是抱持看看就好的心態，畢竟我認為每個人的個性都是持續變化的，可能每隔幾年測一次結果都不盡相同

**Comment by asd99263fg** [comment]
完全不準確，我個人覺得這種個性的問題你如果拿男生女生這種二分法根本是一間愚蠢的問題，尤其是I跟E這兩個的定義非常狹義或者說不清楚的，與其把人類分成兩類，不如想想怎麼樣跟人想處還比較實在，不過我覺得這也不適絕對，畢竟每個人在不同環境會呈現不同個性這個應該是每個人都會遇到的狀況

**Comment by yjpxi** [comment]
我覺得是因應個人當下的情況來決定你是i人還是e人，雖然 MBTI 測驗和其他心理測驗有其局限性，但它們仍然具有一定的參考價值。如果是不了解自己的人，可以通過這些測驗，對自己的性格有初步的了解。

**Comment by MingWei174** [comment]
這種性格測驗是檢視你在測驗當下的心理狀態，但我們人類的心理狀態可能會隨著年齡增長或者是周遭人事物的影響而產生變化，因此可能過個幾年再測一次又是一個截然不同的MBTI，但如果有認真回答測驗中的題目，應該是可以透過這個測驗了解到當下自己真實的性格。

**Comment by Nana1218** [comment]
這種性格測驗非常容易因為當下自身的心理狀態而得出截然不同的結果，並且我認為更重要的是，沒有人是能被完全歸類的，在不同時後展現出的自我都是不一樣的，沒有人會是絕對的活潑，也沒有人會是絕對的內向，當下身邊的環境跟人都會讓人給出不同的反應。

**Comment by kingsophy** [comment]
我認為這些心理測驗對我來說就像是星座一樣，只是對大數據進行分析並得出結果，造成有大部分的人覺得很準，不過卻有一部份的人和敘述不同或甚至相反，因此我都把這些心理測驗我也只是看看而已，真正要認識一個人還是要面對面聊天才行。

**Comment by ds0w0** [comment]
問就是僅供參考，心理測驗只是根據你的回答生成出一個對應的結果，關於信度和效度都十分可議。我自己認為這只是聽供一個大致的方向能夠標籤化自己，而實際上人是不可能像MBTI這樣被二分的，應該呈現光譜或是更多面向，所以想認識一個人不能如此片面。
